doc/howto/storage_volumes.md

```diff
@@ -107,49 +110,49 @@ Instead of attaching a custom volume to an instance as a disk device, you can al
 
-- To use a custom volume to store the image tarballs:
+- To use a custom volume `my-images-volume` to store the image tarballs:
 
-      lxc config set storage.images_volume <pool_name>/<volume_name>
+      lxc config set storage.images_volume=my-pool/my-images-volume
 
 (storage-configure-volume)=
 ## Configure storage volume settings
 
-See the {ref}`storage-drivers` documentation for the available configuration options for each storage driver.
-
-Use the following command to set configuration options for a storage volume:
-
-    lxc storage volume set <pool_name> [<volume_type>/]<volume_name> <key> <value>
-
-The default {ref}`storage volume type <storage-volume-types>` is `custom`, so you can leave out the `<volume_type>/` when configuring a custom storage volume.
+See the {ref}`storage-drivers` documentation for a list of available storage volume configuration options for each driver.
 
-For example, to set the size of your custom storage volume `my-volume` to 1 GiB, use the following command:
+To set the maximum size of custom storage volume `my-volume` to 1 GiB, use the following command:
 
     lxc storage volume set my-pool my-volume size=1GiB
 
-To set the snapshot expiry time for your virtual machine `my-vm` to one month, use the following command:
+The default {ref}`storage volume type <storage-volume-types>` is `custom`, but
+other volume types can be configured by using the `<volume_type>/<volume_name>` syntax.
+
+To set the snapshot expiry time for virtual machine `my-vm` to one month, use the following command:
 
-    lxc storage volume set my-pool virtual-machine/my-vm snapshots.expiry 1M
+    lxc storage volume set my-pool virtual-machine/my-vm snapshots.expiry=1M
 
-You can also edit the storage volume configuration by using the following command:
+You can also edit the storage volume configuration as YAML in a text editor:
 
-    lxc storage volume edit <pool_name> [<volume_type>/]<volume_name>
+    lxc storage volume edit my-pool virtual-machine/my-vm
 
 (storage-configure-vol-default)=
 ### Configure default values for storage volumes
 
 You can define default volume configurations for a storage pool.
-To do so, set a storage pool configuration with a `volume` prefix, thus `volume.<VOLUME_CONFIGURATION>=<VALUE>`.
+To do so, set a storage pool configuration with a `volume` prefix: `volume.<KEY>=<VALUE>`.
 
-This value is then used for all new storage volumes in the pool, unless it is set explicitly for a volume or an instance.
-In general, the defaults set on a storage pool level (before the volume was created) can be overridden through the volume configuration, and the volume configuration can be overridden through the instance configuration (for storage volumes of {ref}`type <storage-volume-types>` `container` or `virtual-machine`).
+This value is used for all new storage volumes in the pool, unless it is explicitly overridden.
+In general, the defaults set at the storage pool level can be overridden through
```

Comment:
we shouldnt overflow lines please

Reply:
this was the guidance from Ruth because new lines are interpreted by the docs engine.

Reply:
FYI - in the Markdown flavor used by the docs (MyST, a superset of CommonMark), a single newline is ignored. This is in contrast to the GitHub-flavored Markdown you might be used to, which renders single newlines. Text that follows a single newline is added to the end of the previous line, without a line break. 

Double newlines are rendered normally. If you want to render a single newline, add either two spaces or a `<br />` tag at the end of the line, followed by a newline.

Reply:
Thanks for the context. All line breaks are on sentence boundaries now.

Reply:
ah thanks for updating me on that @minaelee

Reply:
> Text that follows a single newline is added to the end of the previous line, without a line break.

that was it now I misremembered. Sorry @MggMuggins